apis/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -55,7 +55,8 @@ const (
 	defaultCollectKubernetesEvents bool = true
 
 	// defaultAdmissionControllerEnabled          bool = false
```

Comment:
✂️

Reply:
The commented constants in that file are ones that are ones that are [defaulted in the agent code](https://github.com/DataDog/datadog-operator/blob/37d139a670dfcc0197d3a3d64e759e175dd232e1/apis/datadoghq/v2alpha1/datadogagent_default.go#L15) rather than set in the operator. They were added there as reminders so we don't have to check the agent code every time, but would it be better to remove all of them?

Reply:
Thanks for sharing the context! I didn't know we had them on purpose